activities/FoodChain.activity/activity/activity.info

```diff
@@ -1,6 +1,15 @@
 [Activity]
 name = FoodChain
 activity_version = 4
+license = ASL 2.0
 bundle_id = org.olpcfrance.FoodChain
 exec = sugar-activity-web
 icon = activity-icon
+summary = FoodChain is a pedagogical game to learn name of animals.
+tags = Science;Game
+repository = https://github.com/llaske/FoodChain
+url = http://activities.sugarlabs.org/en-US/sugar/addon/4612
```

Comment:
As already mentioned on https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/c7a7dcea7d93d76299aa1267e55b8b5053553b8f/src/sugar3/bundle/__init__.py#L59

The URL tag contains user documentation if available on https://help.sugarlabs.org